internal/freeport/freeport.go

```diff
@@ -0,0 +1,92 @@
+// The MIT License
+//
+// Copyright (c) 2022 Temporal Technologies Inc.  All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2024 Temporal Technologies Inc.  All rights reserved.
```

Reply:
I _believe_ the server license lint check requires 2022. Let me double-check.

Reply:
Huh, I must have imagined that; fixed!